CMSgov/design-system
# [NO-TICKET] Clean up Storybook build dir

## Summary

Don't copy more files than we need to into the Storybook build

- Stops copying the whole `components` folder into Storybook, which adds a lot of files to our deploys
- Instead explicitly copies the core `images` and `fonts` folders like we do for the child design systems

## How to test

1. Remove any local copies of `storybook-static`
2. Run `yarn build:storybook`
3. There should be no `components` folder anymore

## Checklist
